### ISSUE TITLE
[Feature]can I customize the length of test dataset?

### ISSUE THREAD

**Issue by finninmunich** [open-mmlab/mmdetection3d#2853]
### What is the feature?

Hi! 
First of all, thanks for your excellent work!
I am testing the quality of some images generated by bounding box layout. To be more specific, I use the image layout generated fron nuscenes validation set to generate some fake image and I want to run a pretrained detection model on it. However, if I simply use tools/test.py for evaluation, I check the codes  and it will create an object name NuscensEval and load gt from it for calculating metrics. My problem is that this object will load all samples from nuscenes validation set, which takes too much time for evaluation. Is ther any config or codes I can change so that I can use tools/test.py to evluate my model on some samples in the validation set? (such as the first 10%) In that case, I can simply create a .pkl file which contains only some sample from the val set and use it for validation.
Thanks in advance!
